schema/data_feed/tracker.fbs

```diff
@@ -9,34 +9,50 @@ namespace solarxr_protocol.data_feed.tracker;
 /// Trackers may be synthetic/computed or instead part of an actual hardware device.
 /// There can be multiple trackers per hardware device.
 table TrackerData {
-   	tracker_id: solarxr_protocol.datatypes.TrackerId;
+    tracker_id: solarxr_protocol.datatypes.TrackerId;
 
-	info: TrackerInfo;
+    info: TrackerInfo;
 
-	status: solarxr_protocol.datatypes.TrackerStatus;
+    status: solarxr_protocol.datatypes.TrackerStatus;
 
-	// Everything here is optional and controlled by `TrackerDataMask`
-	rotation: solarxr_protocol.datatypes.math.Quat;
-	/// Position, in meters
-	// TODO: With respect to what space?
-	position: solarxr_protocol.datatypes.math.Vec3f;
-	/// Raw rotational velocity, in euler angles
-	raw_rot_vel: solarxr_protocol.datatypes.math.Vec3f;
-	/// Raw translational acceleration, in m/s^2
-	raw_trans_accel: solarxr_protocol.datatypes.math.Vec3f; // trans? BASED
-	/// Temperature in degrees celsius
-	temp: solarxr_protocol.datatypes.Temperature;
+    // Everything here is optional and controlled by `TrackerDataMask`
+
+    /// Raw sensor rotation
+    rotation: solarxr_protocol.datatypes.math.Quat;
+    /// Position, in meters
+    // TODO: With respect to what space?
+    position: solarxr_protocol.datatypes.math.Vec3f;
+    /// Raw angular velocity, in euler angles, rad/s
+    raw_angular_velocity: solarxr_protocol.datatypes.math.Vec3f;
+    /// Raw acceleration, in m/s^2
+    raw_acceleration: solarxr_protocol.datatypes.math.Vec3f;
+    /// Temperature, in degrees celsius
+    temp: solarxr_protocol.datatypes.Temperature;
+    /// Acceleration without gravity, in m/s^2
+    linear_acceleration: solarxr_protocol.datatypes.math.Vec3f;
+    
+    /// Reference-adjusted rotation for IMU trackers (VR HMD yaw is used as a reset reference).
+    /// Includes: mounting orientation, full, quick and mounting reset adjustments.
+    /// This rotation can be used to reconstruct a skeleton pose using forward kinematics.
+    rotation_reference_adjusted: solarxr_protocol.datatypes.math.Quat;
+    /// Zero-reference-adjusted rotation for IMU trackers (identity quaternion is used as a reset reference).
+    /// Includes: only full and quick reset adjustments.
+    /// This rotation can be used in visualizations for IMU debugging.
+    rotation_identity_adjusted: solarxr_protocol.datatypes.math.Quat;
```

Comment:
when you say "identity" I assume you mean the identity quat. In which case, is this always the identity quat? I'm assuming its not because then sending data would never change and be useless. So what is it actually? I still don't understand what a "zero-reference-adjusted" rotation is.

Also, what does it mean for it to include both full and quick reset adjustments? This wording is pretty vague and not specific enough - what does "include" here mean? What are you referring to when you say "quick reset adjustments"? Is it the act of resetting that you're referring to, or a quaternion saved from reset, or something else?

Reply:
You can think of that quaternion as “the rotation (from the identity quat) since last reset”

Reply:
"rotation with respect to identity quat" identity quat in what frame? sensor frame or reset frame?

Reply:
Can resolve this if we add to the docs that these fields are SlimeVR-specific and exclusively computed by our server? This doesn't seem like something that can be easily explained without writing a few paragraphs.

Reply:
yes that is fine!

Reply:
Okay now it also contains some sort of explanation.